tests/dragonfly/set_test.py

```diff
@@ -23,3 +23,21 @@ async def test_sscan_regression(df_factory: DflyInstanceFactory):
     # Takes 3 seconds
     res = await client.execute_command("SLOWLOG GET 100")
     assert res == []
+
+
+@pytest.mark.asyncio
+async def test_sinter_regression(df_factory: DflyInstanceFactory):
+    df = df_factory.create(proactor_threads=4, shard_round_robin_prefix="prefix-")
```

Comment:
I forced the largest set to be picked via the `shard_round_robin` flag

Reply:
yeah it was an overkill IMO

Reply:
+1 to c++ test. we should not use pytests for testing a command logic.